linux_os/guide/system/accounts/accounts-restrictions/account_expiration/ensure_shadow_group_empty/ansible/shared.yml

```diff
@@ -0,0 +1,5 @@
+# platform = multi_platform_all
+
+- name: Ensure interactive local users are the owners of their respective initialization files
+  ansible.builtin.shell:
+    cmd: sed -ri 's/(^shadow:[^:]*:[^:]*:)([^:]+$)/\1/' /etc/group
```

Comment:
You might be able to do this with `lineinfile` vs `shell` and that would be preferred.

Reply:
thx should be fixed in [9e95276](https://github.com/ComplianceAsCode/content/pull/12921/commits/9e95276c6f8d0e98f4fa226f66e705a5323176f3) :bow: